$OpenBSD: patch-config,v 1.3 2017/05/25 14:39:24 sthen Exp $

Don't misdetect OpenBSD/i386 as a.out when /usr/lib/libc.so.XX.Y.a
is present (for /etc/rc rebuildlibs).

Index: config
--- config.orig
+++ config
@@ -762,6 +762,7 @@ case "$GUESSOS" in
   sparc64-*-*bsd*)	OUT="BSD-sparc64" ;;
   ia64-*-*bsd*)		OUT="BSD-ia64" ;;
   amd64-*-*bsd*)	OUT="BSD-x86_64" ;;
+  *86*-*-openbsd)	OUT="BSD-x86-elf" ;;
   *86*-*-*bsd*)		# mimic ld behaviour when it's looking for libc...
 			if [ -L /usr/lib/libc.so ]; then	# [Free|Net]BSD
 			    libc=/usr/lib/libc.so
